core/http-auth-spi/src/main/java/software/amazon/awssdk/http/auth/spi/IdentityProviderConfiguration.java

```diff
@@ -26,11 +29,30 @@
  * configured on the SDK.
  */
 @SdkPublicApi
-@FunctionalInterface
```

Comment:
In making it a ToCopyableBuilder the interface also has `toBuilder()` to implement now, so it is no longer a `@FunctionalInterface`.